client/interfaces.go

```diff
@@ -83,6 +83,7 @@ type InstanceServer interface {
 	CreateCertificate(certificate api.CertificatesPost) (err error)
 	UpdateCertificate(fingerprint string, certificate api.CertificatePut, ETag string) (err error)
 	DeleteCertificate(fingerprint string) (err error)
+	CreateClientJoinToken(certificate api.CertificatesPost) (op Operation, err error)
```

Comment:
`CreateCertificateToken` and it should probably ensure that `certificate.Token` is set to True.